packages/emails/templates/_base-email.ts

```diff
@@ -1,5 +1,5 @@
 import { decodeHTML } from "entities";
-import nodemailer from "nodemailer";
+import {createTransport} from "nodemailer";
```

Comment:
@tony83033  are there any benefits of importing only createTransport ?

Reply:
I think tree shaking works better this way.